cmake/spectra-config.cmake.in

```diff
@@ -1,5 +1,7 @@
 @PACKAGE_INIT@
 
+find_package (Eigen3 CONFIG)
```

Comment:
It should be `find_dependency` in a CMake config file, rather than `find_package`: https://cmake.org/cmake/help/latest/module/CMakeFindDependencyMacro.html

Reply:
Thanks. Already fixed by #140.